backend/api/contacts/src/main/java/co/airy/core/contacts/dto/Contact.java

```diff
@@ -99,12 +99,12 @@ public Address merge(Address address) {
             }
 
             return this.toBuilder()
-                    .addressLine1(Optional.ofNullable(this.getAddressLine1()).orElse(address.getAddressLine1()))
```

Comment:
This was the wrong way around. Third time's a charm 😄

Reply:
🙈